adhoc-build.sh

```diff
@@ -0,0 +1,3 @@
+
```

Comment:
Handy script for adhoc builds:

> ./adhoc-build ~/Desktop

Will deposit a timestamped IPA on your desktop so long as you have the private key installed and your Xcode setup for managing profiles, etc.